### DISCUSSION TITLE
Curl error while SSL Verification

### DISCUSSION THREAD

**Discussion by espiritulibre** [EasyEngine/easyengine#1692] [Support Requests]
Hello,

I just tried to install Easyengine on my clean Ubuntu 20.04 System but somehow I cant create SSL certificates and I dont know what the exact problem is .

I just got this error message 

`
Moved /var/www/htdocs/wp-config.php to /var/www/wp-config.php successfully
Checking and verifying site-up status. This may take some time.
.
Installing WordPress site.
Success: https://mydomain.com has been created successfully!
Starting SSL verification.
Warning: cURL error 28: Failed to connect to mydomain.com port 80: Connection timed out (see http://curl.haxx.se/libcurl/c/libcu                rl-errors.html)
Warning: Initiating clean-up.
[mydomain.com] Docker Containers removed.
[mydomain.com] site root removed.
Success: Site mydomain.com deleted.
Report bugs here: https://github.com/EasyEngine/site-type-wp
`

**Comment by mhbackpacker** [comment]
I have the same issue! Is there a setup guide I am missing? I followed this setup guide here: https://www.cloudbooklet.com/how-to-install-or-migrate-wordpress-with-easyengine-setup/

**Comment by vishalmukadam** [comment]
Hello! @espiritulibre 

While creating site use the command with --ssl=le and see if it works.

Considering you already have installed EE to your machine.

Now while creating site, you generally use command,

sudo ee site create yoursitename.com --type=wp **--type=le**

You can directly copy-paste the above command.

You can refer to the guide here: https://easyengine.io/commands/site/create/

I hope that will help.


Thanks!
